osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -854,28 +855,31 @@ private void bindBindables()
         /// Will immediately run filter operations if required.
         /// </summary>
         /// <returns>Whether a transfer occurred.</returns>
-        private bool transferRulesetValue()
+        private bool transferRulesetValue(bool carouselLoaded = true)
```

Comment:
This is bad and cannot exist

Reply:
Wha would be a better way of blocking usage without making the carousel nullable?

Reply:
Check my better solution.